### ISSUE TITLE
Is this package still maintained?

### ISSUE THREAD

**Issue by jonaswre** [microsoft/azure-devops-extension-api#72]
Hi, 
I was wondering if the package is still maintained? 
There is still a lot of functionality missing from the RestAPI
Last commit from 10 month ago... And Pull Request ready to merge for months... 

If this is dead anybody looking for a community effort to implement missing functionality?

**Comment by jan-machacek** [comment]
Exactly, it's sad that tool for development support is so out of date. A lot of API is missing or is in old version. DevOps UI components is exactly same like this with no support.

**Comment by jeffpriz** [comment]
yeah, this is maddening! No updates, no support, no nothing, just dropped it and stopped updating... There are APIs I need to call that this API client hasn't been updated to allow..  
I guess it's open source so we can try to keep it up.. but that's not what I bargained for in trying to make an extension for AzDevOps

**Comment by jonaswre** [comment]
I've created a repo and setup the npm part.
If you want to append any request. Please do so. 

https://github.com/byndit/azure-devops-extension-api
https://www.npmjs.com/package/@byndit/azure-devops-extension-api

If any Microsoft employee ever comes back to here, please let us know when you have implemented the changes that where made.

**Comment by BearsPunch** [comment]
> I've created a repo and setup the npm part. If you want to append any request. Please do so.
> 
> https://github.com/byndit/azure-devops-extension-api https://www.npmjs.com/package/@byndit/azure-devops-extension-api
> 
> If any Microsoft employee ever comes back to here, please let us know when you have implemented the changes that where made.

Looks like you also dropped support of your mirror :(

**Comment by jonaswre** [comment]
We currently have the needed features. If you need anything more please create a PR I'll definitely look into it and accept if it looks good.
